### ISSUE TITLE
PUT: handle single boolean value

### ISSUE THREAD

**Issue by nicolidin** [typicode/json-server#1248]
Hello, 

json-server does not allow to put a single value request payload in a PUT route.
``` typescript
axios.put(`/articles/${payload.postMortem.id}/isReleased`, true)
```
json-server will say "SyntaxError: Unexpected token t in JSON at position 0 at JSON.parse (<anonymous>)"

If we wrap the boolean payload in an object like: {val: true}, it will work.

Hope you can do something cause it's pretty important!

(version:
   "json-server": "^0.16.1")

**Comment by ThisUsernameIsAllreadyTaken** [comment]
This won't change I'm pretty sure. I read about this issue many times.

**Comment by ThisUsernameIsAllreadyTaken** [comment]
Actually sorry, I think I misunderstood.

Can't you just PATCH instead of PUT on route ending with id?

`axios.patch(`/articles/${payload.postMortem.id}`, {isReleased: true})`

**Comment by nicolidin** [comment]
No, the idea is not to find a workaround and especially not to change my spec

**Comment by ThisUsernameIsAllreadyTaken** [comment]
Just wanted to help, because you say it's so important and this seems to be the intended way.

If you can't change your spec no matter what, I think you can write a middleware, that does the trick.
I understand that you don't want a workaround, but since the last commit is 6 months ago, I wouldn't put too much hope in @typicode fixing it.

cheers
